### ISSUE TITLE
[Bug]: TypeError: visitFn.call is not a function

### ISSUE THREAD

**Issue by tkvw** [ticruz38/graphql-codegen-svelte-apollo#25]
Using `graphql` 16.x I get the error: 
```
Client path is not present in config
  ✔ Parse configuration
  ❯ Generate outputs
    ❯ Generate src/generated.ts
      ✔ Load GraphQL schemas
      ✔ Load GraphQL documents
      ✖ Generate
        → visitFn.call is not a function


 Found 1 error

  ✖ src/generated.ts
    TypeError: visitFn.call is not a function
        at visit (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/graphql@16.0.1/node_modules/graphql/language/visitor.js:200:21)
        at Object.plugin (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/graphql-codegen-svelte-apollo@1.0.1_graphql@16.0.1/node_modules/graphql-codegen-svelte-apollo/src/index.js:30:49)
        at executePlugin (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:38:35)
        at async /home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:167:24
        at async Promise.all (index 2)
        at async Object.codegen (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:157:20)
        at async process (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:1003:56)
        at async Promise.all (index 0)
        at async /home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:1010:37
        at async Task.task (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:789:17)
    TypeError: visitFn.call is not a function
        at visit (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/graphql@16.0.1/node_modules/graphql/language/visitor.js:200:21)
        at Object.plugin (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/graphql-codegen-svelte-apollo@1.0.1_graphql@16.0.1/node_modules/graphql-codegen-svelte-apollo/src/index.js:30:49)
        at executePlugin (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:38:35)
        at async /home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:167:24
        at async Promise.all (index 2)
        at async Object.codegen (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+core@2.3.0_graphql@16.0.1/node_modules/@graphql-codegen/core/index.js:157:20)
        at async process (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:1003:56)
        at async Promise.all (index 0)
        at async /home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:1010:37
        at async Task.task (/home/dennie/src/github.com/tkvw/svelte/sveltekitts-pokedex/node_modules/.pnpm/@graphql-codegen+cli@2.2.2_graphql@16.0.1+typescript@4.4.4/node_modules/@graphql-codegen/cli/bin.js:789:17)

```

**Comment by ticruz38** [comment]
Could you share your config file?

**Comment by mattwilson1024** [comment]
I'm getting the same error with graphql `16.x`. My `codegen.yml` is below:

```
overwrite: true
schema: "http://localhost:3001/graphql"
documents: ["src/**/*.graphql", "!src/generated"]
generates:
  src/generated/graphql.ts:
    plugins:
      - "typescript"
      - "typescript-operations"
      - "graphql-codegen-svelte-apollo"
    config:
      clientPath: src/lib/apollo-client.ts
      asyncQuery: true
  src/generated/schema.graphql:
    plugins:
      - schema-ast
    config:
      sort: true
```

and `src/lib/apollo-client.ts` looks like this:

```
import { ApolloClient, InMemoryCache } from '@apollo/client/core';

const cache = new InMemoryCache({
  addTypename: true,
});

export default new ApolloClient({
  uri: 'http://localhost:3001/graphql',
  cache,
});
```

It works if I downgrade to graphql 15.x, but not with 16.x.

**Comment by ticruz38** [comment]
ok, I have not tested with graphql@16, thanks for reporting, I'll try to fix it ASAP

**Comment by jycouet** [comment]
Same for me.
I guess that it has to do with this PR: https://github.com/dotansimha/graphql-code-generator/pull/6892

Maybe a dep update will fix the issue

**Comment by ticruz38** [comment]
I just published a new version@1.1.0, if you could take a look and let me know.
graphql@16 slightly changed their methods, that's why it was incompatible
